modules/machineset-creating-efa-options.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-creating-efa-options_{context}"]
+= Creating machines that use an AWS Elastic Fiber Adapter
```

Comment:
This needs updating, it's `Elastic Fabric Adapter`, ref, https://aws.amazon.com/hpc/efa/